tqdm/_tqdm.py

```diff
@@ -342,7 +353,7 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
                 bar_length, frac_bar_length = divmod(
                     int(frac * N_BARS * 10), 10)
 
-                bar = '#' * bar_length
+                bar = '█' * bar_length
```

Comment:
I just replaced "#" with this sign because it looks better on my system, didn't know how it works on another systems